index.html

```diff
@@ -0,0 +1,189 @@
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button class="btn" type="button">See Project</button>
+                </form>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button class="btn" type="button">See Project</button>
+                </form>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button class="btn" type="button">See Project</button>
+                </form>
+            </div>
+        </div>
+        <section class="about">
+            <div class="about-text">
+                <div class="about-section">
+                    <h3>About me</h3>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                        of
+                        my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                        to
+                        contact me.</p>
+                    <button  class="btn" type="button">Get My Resume</button>
+                </div>
+                
+            </div>
+            <div class="language">
+                <ul>
+                    <li>
+                        <h4>Language</h4>
+                        <ul>
+                            <li>Javascript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    </li>
```

Comment:
- [ ] You have done a great job implementing changes, but I still can think that the styling applied for this section does not match the design, please stick to the template given in the Figma, consider applying changes gave by the previous review, please. see images below ⬇️ 

- [ ] Your Design 🏃🏻 

<img width="320" alt="his" src="https://user-images.githubusercontent.com/79058364/153563510-496c144a-93c2-4f6d-9780-1e23248ef1b6.png">

- [ ] Figma one ❤️ 

<img width="303" alt="rv" src="https://user-images.githubusercontent.com/79058364/153563584-a8edbb5a-8dba-4b62-a8f0-3de4fc140889.png">